### ISSUE TITLE
Increase instance size to improve stability

### ISSUE THREAD

**Issue by KlaasH** [PublicMapping/districtbuilder#1311]
Per discussion in the meeting just now, the first thing it makes sense to try, to help with the recent uptick in app task instability, is to go back to a larger EC2 instance size.

While I was looking through the Terraform config and variables to do that, a few other things caught my eye as well.  So here are the deployment changes I'm proposing to make.  I don't think any of them require changes to the Terraform code, just variable changes, so this is an annotated diff of the updated production tfvars file vs. the current one:
- Increase the instance size:
  ```diff
  -container_instance_type    = "r6a.xlarge"
  +container_instance_type    = "r6a.2xlarge"
  ```
- Note: the total amount of memory allocated to the task is calculated [in `container_service.tf`](https://github.com/PublicMapping/districtbuilder/blob/911fdcbb0ecbc5b11921c00a9adba13eb02fd49a/deployment/terraform/container_service.tf#LL187C12-L187C12) based on the instance type, so there's no variable to update for that.  I didn't change `container_instance_reserved_memory`, though we should be sure to check, once a new instance is up, that we've reserved enough for the task to actually get placed on the instance.
- Increase storage space, since that's been causing issues lately (see [this Slack thread](https://element84.slack.com/archives/C04UJQ8VAV8/p1686671392565419)). If everything is running smoothly it won't get used, but having a little more margin for error right now seems worthwhile. I also switched the storage type from gp2 to gp3. gp3 is a drop-in replacement, should be slight faster (unlikely to matter, but certainly won't hurt) and currently costs 20% less per GB than gp2 storage.
  ```diff
  -container_instance_root_block_device_type        = "gp2"
  -container_instance_root_block_device_size        = 64
  +container_instance_root_block_device_type        = "gp3"
  +container_instance_root_block_device_size        = 128
  ```
- Adjust the tfvars to reflect the changes made manually ([as described on Slack](https://element84.slack.com/archives/C04UJQ8VAV8/p1686666373199179?thread_ts=1686615955.999949&cid=C04UJQ8VAV8)) to make the health checks more forgiving:
  ```diff
  -target_group_health_check_healthy_threshold   = 3
  +target_group_health_check_healthy_threshold   = 2
   target_group_health_check_interval            = 30
  -target_group_health_check_timeout             = 3
  -target_group_health_check_unhealthy_threshold = 2
  +target_group_health_check_timeout             = 5
  +target_group_health_check_unhealthy_threshold = 3
  ```

**Comment by KlaasH** [comment]
The first bullet originally included bumping the CPU allocation, too:
  ```diff
  -container_instance_app_cpu = 4096
  +container_instance_app_cpu = 8192
  ```
but actually since the app [calculates how many workers to run based on the amount of CPU available](https://github.com/PublicMapping/districtbuilder/blob/develop/src/server/src/common/constants.ts#L14), we don't want to use the full CPU allocation.  That would give us more workers with the same amount of memory each, but we want to run the same number of workers and have them each get more memory.

**Comment by KlaasH** [comment]
Until we address [issue #1313](https://github.com/PublicMapping/districtbuilder/issues/1313), there's not actually an advantage to running bigger instances in terms of keeping workers from running out of memory, and actually running fewer, bigger tasks is slightly worse for availability than running more, smaller ones.
So I'm going to set `container_instance_type` back to `r6a.xlarge` for the new deployment I'm about to do once I've merged PR #1315.

It might be fine to turn the instance root device size back down, too, but it seems better to leave it higher until we've done https://github.com/PublicMapping/districtbuilder/issues/1310.
